### PR TITLE
fix: remove lazy for large content paint images

### PR DESCRIPTION
**Pull request by asvishnyakov**
Description:
https://virtocommerce.atlassian.net/browse/ST-4743

After this PR will be merged, use `BREAKPOINTS` from `import { BREAKPOINTS } from "@/core/constants"` instead of `breakpointsTailwind` from `import { breakpointsTailwind } from "@vueuse/core"`.

--

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.37.0-pr-677-ac8c-ac8ce4b7.zip

### CODE REVIEW

File: client-app/shared/catalog/components/product-card-list.vue [hunk 1]
```diff
@@ -167,14 +167,15 @@ const props = defineProps<IProps>();
...
+  lazy: boolean;
   openInNewTab?: boolean;
 }
 
 const link = computed<RouteLocationRaw>(() => getProductRoute(props.product.id, props.product.slug));
 const target = computed<string>(() => getLinkTarget(props.openInNewTab));
 const isDigital = computed<boolean>(() => props.product.productType === ProductType.Digital);
 const properties = computed(() =>
-  Object.values(getPropertiesGroupedByName(props.product.properties ?? [], PropertyType.Product)).slice(0, 3)
+  Object.values(getPropertiesGroupedByName(props.product.properties ?? [], PropertyType.Product)).slice(0, 3),
```
Comment (NaMax66) [inline client-app/shared/catalog/components/product-card-list.vue:178]: do wo really need this comma?

File: client-app/shared/static-content/components/slider.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
       <Swiper :slides-per-view="1" class="w-full" :modules="modules" :navigation="navigationOptions">
         <SwiperSlide v-for="(item, index) in model.slides" :key="index" class="text-center">
           <div>
-            <VcImage :src="item.image" class="place-items-center" lazy />
+            <VcImage :src="item.image" class="place-items-center" :lazy="index > 0" />
```
Comment (NaMax66) [inline client-app/shared/static-content/components/slider.vue:9]: rename index to blockOrder. 
it should be first

File: client-app/ui-kit/components/atoms/image/vc-image.vue [hunk 3]
```diff
@@ -14,35 +14,23 @@
...
-  sizeSuffix: {
-    type: String as PropType<"sm" | "md" | "lg">,
-    validator: (value: string) => ["sm", "md", "lg"].includes(value),
-  },
+  sizeSuffix?: "sm" | "md" | "lg";
+}
+
+const props = withDefaults(defineProps<IProps>(), {
+  src: "",
+  fallbackSrc: "/static/images/common/no-image.svg",
```
Comment (NaMax66) [inline client-app/ui-kit/components/atoms/image/vc-image.vue:33]: move to const

File: client-app/ui-kit/components/atoms/image/vc-image.vue [hunk 4]
```diff
@@ -86,6 +74,6 @@ watch(
   () => {
     fallbackEnabled.value = false;
     originalEnabled.value = false;
-  }
+  },
```
Comment (NaMax66) [inline client-app/ui-kit/components/atoms/image/vc-image.vue:77]: aaa

File: client-app/shared/catalog/components/product-card-list.vue (hunk 1 above)
Comment (asvishnyakov) [reply client-app/shared/catalog/components/product-card-list.vue:178]: It's from linting

File: client-app/shared/static-content/components/slider.vue (hunk 2 above)
Comment (asvishnyakov) [reply client-app/shared/static-content/components/slider.vue:9]: Why? It's already used everywhere and is default https://vuejs.org/api/built-in-directives.html#v-for

File: client-app/ui-kit/components/atoms/image/vc-image.vue (hunk 3 above)
Comment (asvishnyakov) [reply client-app/ui-kit/components/atoms/image/vc-image.vue:33]: Why? It's default prop value

Comment (asvishnyakov) [reply client-app/ui-kit/components/atoms/image/vc-image.vue]: Fixed

**Review by NaMax66 — APPROVED** [review]

**Pull request merged**